data/src/main/java/com/microsoft/azure/kusto/data/Utils.java

```diff
@@ -57,15 +56,14 @@ private Utils() {
         // Hide constructor, as this is a static utility class
     }
 
-    static String post(CloseableHttpClient httpClient, String url, String payload, InputStream stream, long timeoutMs,
-      Map<String, String> headers, boolean leaveOpen) throws DataServiceException, DataClientException {
-        URI uri = parseUriFromUrlString(url);
+    static String post(CloseableHttpClient httpClient, String urlStr, String payload, InputStream stream, long timeoutMs, Map<String, String> headers, boolean leaveOpen) throws DataServiceException, DataClientException {
```

Comment:
Other than the rename of the parameter, there are no actual changes to this file, right?